src/assets/images/flags/it.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we change the flags back to grayscale? Full colors are too bold and too eye-catching.